libr/core/config.c

```diff
@@ -1805,7 +1805,7 @@ R_API int r_core_config_init(RCore *core) {
 	SETCB("io.sectonly", "false", &cb_iosectonly, "Only read from sections (if any)");
 	SETI("io.buffer.from", 0, "Lower address of buffered cache");
 	SETI("io.buffer.to", 0, "Higher address of buffered cache");
-	SETCB("io.cache", "false", &cb_iocache, "Enable cache for io changes");
+	SETCB("io.cache", "true", &cb_iocache, "Enable cache for io changes");
```

Comment:
Wouldn't that break r2 -w?
Also, if caches are flushed, will the binary suddenly have changes that were done implicitly?